File: mainActivity/src/main/java/com/sierrawireless/avphone/task/DeleteSystemResult.java
package com.sierrawireless.avphone.task;

import net.airvantage.model.AvError;
import net.airvantage.model.User;

public class DeleteSystemResult {

    private AvError error;

    private User user;

    DeleteSystemResult(AvError error) {
        this.error = error;
    }

    DeleteSystemResult(final User user) {
        this.user = user;
    }

    public boolean isError() {
        return error != null;
    }

    public User getUser() {
        return user;
    }

    public AvError getError() {
        return error;
    }
}

File: mainActivity/src/main/java/com/sierrawireless/avphone/task/DeleteSystemResult.kt
package com.sierrawireless.avphone.task

import net.airvantage.model.AvError
import net.airvantage.model.User

class DeleteSystemResult {

    var error: AvError? = null

    var user: User? = null

    val isError: Boolean
        get() = error != null

    internal constructor(error: AvError) {
        this.error = error
    }

    internal constructor(user: User) {
        this.user = user
    }
}
